docs/content/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s.md

```diff
@@ -0,0 +1,129 @@
+    weight: 20
+isTocNested: true
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-yp" class="nav-link">
+      <i class="fas fa-cloud"></i>
+      Default
+    </a>
+  </li>
+
+  <li>
+    <a href="/latest/yugabyte-platform/administer-yugabyte-platform/back-up-restore-k8s" class="nav-link active">
+      <i class="fas fa-cubes" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+</ul>
+
+Yugabyte Platform installations include configuration settings, certificates and keys, and other components required for orchestrating and managing YugabyteDB universes.
+
+You can use the Yugabyte Platform backup script to back up an existing Yugabyte Platform server and restore it, when needed, for disaster recovery or migrating to a new server.
+
+## Back up a Yugabyte Platform server
+
+1. Copy the the Yugabyte Platform backup script (`yb_platform_backup.sh`) from the yugabyte-db repository to your local workstation using the following wget command:
+
+    ```sh
+    wget https://github.com/yugabyte/yugabyte-db/blob/master/managed/devops/bin/yb_platform_backup.sh
+    ```
+
+1. Verify that the machine performing the backup operation can access the Yugabyte Platform kubernetes pod instance:
+
+    ```sh
+    kubectl exec --namespace <kubernetes_namespace> -it <kubernetes_pod> -c yugaware -- cat /opt/yugabyte/yugaware/README.md
+    ```
+
+    <br/>
+
+    The Yugabyte Platform console is used in a highly available mode and orchestrates and manages YugabyteDB universes, or clusters, on one or more regions (across public cloud and private on-premises data centers). For more details, refer to the [Yugabyte Platform overview](https://docs.yugabyte.com/latest/yugabyte-platform/overview/).
+
+    <br/>
+
+    * `k8s_namespace` specifies the kubernetes namespace where the Yugabyte Platform pod is running.
+    * `k8s_pod` specifies the name of the Yugabyte Platform kubernetes pod.
+
+1. Run the `yb_platform_backup.sh` script using the `backup` command:
+
+    ```sh
+    ./yb_platform_backup.sh backup --output <output_path> --k8s_namespace <kubernetes_namespace> --k8s_pod <kubernetes_pod> [--exclude_releases --verbose]
+    ```
+
+    * `backup` is the command to run the backup of the Yugabyte Platform server.
+    * `output_path` specifies the location for the output backup archive.
+    * `k8s_namespace` specifies the Kubernetes namespace in which the Yugabyte Platform pod is running.
+    * `k8s_pod` specifies the name of the Yugabyte Platform Kubernetes pod.
+    * `exclude_releases` excludes Yugabyte releases from the backup archive.
+    * `verbose` prints debug output.
+
+    {{< note title="Prometheus metrics data" >}}
+
+Backup and restore of Prometheus metrics data is not currently supported.
+
+    {{< /note >}}
+
+1. Verify that the backup `.tar` file, with the correct timestamp, is in the specified output directory.
```

Comment:
Reword (.tar -> .tar.gz) (is in -> is created in):
1. Verify that the backup `.tar.gz` file, with the correct timestamp, is created in the specified output directory.

Reply:
Fixed here and in one other spot...